easybuild/easyblocks/n/numpy.py

```diff
@@ -244,8 +245,12 @@ def sanity_check_step(self, *args, **kwargs):
         }
         custom_commands = [
             ('python', '-c "import numpy"'),
-            ('python', '-c "import numpy.core._dotblas"'),  # _dotblas is required for decent performance of numpy.dot()
         ]
+        if LooseVersion(self.version) < LooseVersion("1.10"):
+            # _dotblas is required for decent performance of numpy.dot()
+            custom_commands.append (('python', '-c "import numpy.core._dotblas"'))
+        else:
+            custom_commands.append (('python', '-c "from numpy.core.multiarray import dot"'))
```

Comment:
I just checked, the list of stuff you can import from `numpy.core.multiarray` is identical between a properly optimised installation, and a broken one, so we need another way to check.